src/test/java/core/basesyntax/FruitDataValidatorCsvImplTest.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+import validators.FruitDataValidator;
+import validators.FruitDataValidatorCsvImpl;
+
+public class FruitDataValidatorCsvImplTest {
+    private static final FruitDataValidator VALIDATOR = new FruitDataValidatorCsvImpl();
```

Comment:
```suggestion
    private static final FruitDataValidator validator = new FruitDataValidatorCsvImpl();
```
якщо ми не робимо клас deeply immutable то неймінг повинен бути як простого класу (String має такий неймінг так як це immutable class)